library/x509.c

```diff
@@ -1508,11 +1508,15 @@ int mbedtls_x509_info_subject_alt_name(char **buf, size_t *size,
             case MBEDTLS_X509_SAN_DNS_NAME:
             case MBEDTLS_X509_SAN_RFC822_NAME:
             {
-                char * dns_name = "dNSName";
-                char * rfc822_name = "rfc822Name";
-
-                ret = mbedtls_snprintf(p, n, "\n%s    %s : ", prefix,
-                        san.type == MBEDTLS_X509_SAN_DNS_NAME ? dns_name : rfc822_name);
+                const char *dns_name = "dNSName";
+                const char *rfc822_name = "rfc822Name";
+
+                ret = mbedtls_snprintf(p,
```

Comment:
Minor: Isn't this a bit too much? doesn't p and n really not fit in one line?

Reply:
I think by default the code style script puts each one on a separate line but it'll be happy if you have e.g.:
```c
ret = mbedtls_snprintf(p, n,
                       "\n%s %s : "
                       ...)
```
Just as long as any line continuations are aligned after the opening paren.

Reply:
I have change line to have `p` and `n` in same line, but it doesn't looks good if we need to correct style script.